docs/k8s.md

```diff
@@ -1,47 +1,151 @@
+
+First, we need to edit a line in each unit file of the group to make the unit use the latest tag:
+
+`Environment="IMAGE=giantswarm/k8s-kubelet:1.2.2"` in `k8s-node-kubelet@.service`
+
+and
+
+`Environment="IMAGE=giantswarm/k8s-proxy:1.2.2"` in `k8s-node-proxy@.service`
+
+Now, a single command lets us update the whole k8s-node group
+
+```
+$ inagoctl update k8s-node --max-growth=0 --min-alive=2 --ready-secs=60
+2016-04-14 15:45:22.988 | INFO     | context.Background: Succeeded to update 3 slices for group 'k8s-node': [045 3a9 79b].
+```
+
+And we're done! After a while, all nodes will be updated and another `kubectl describe nodes` will show that we’re now running 3 nodes with kubelet and proxy version 1.2.2 deployed.
+
+Looking closely at the update command above we can inspect some of its options. `--max-growth=0` tells Inago to not start any additional instances, which means it has to remove an instance to be able to start a new one. In this example this is due to the fact that we don't have any free machine left. `--min-alive=2` tells Inago to keep at least two instances of the group alive during the update process. the `ready-secs` flag determines how long Inago waits between rounds of updating. This gives the new node a bit of time to start replicating that pod we scheduled.
+
+Watching `kubectl get deployments` during the update process, we would see that available instances of our hello-node pod shortly go down to 2, while a Kubernetes node is being updated and then quickly come up again to 3.
+
+### Updating the Master
+
+Now sadly, we didn't use an HA deployment of the Kubernetes master, so Inago's update command isn't of help much here. Accordingly, we will have a short downtime of our API while we update this group. Luckily, Kubernetes nodes and their pods don't rely on the API server that much and will usually stay running while we update the master.
+
+Again we need to update our unit files to use the newest image. After that a short series of commands will update the group.
 
-To validate that the pods indeed got started, we can take a look at the pods.
 ```
-$ kubectl -s 172.17.8.102:9090 get pods
-NAME             READY     STATUS    RESTARTS   AGE
-my-nginx-0uhad   0/1       Pending   0          13s
-my-nginx-zuztz   0/1       Pending   0          13s
-nginx            1/1       Running   1          1h
+$ inagoctl stop k8s-master
+2016-04-14 16:32:08.465 | INFO     | context.Background: Succeeded to stop group 'k8s-master'.
+
+$ inagoctl destroy k8s-master
+2016-04-14 16:32:27.384 | INFO     | context.Background: Succeeded to destroy group 'k8s-master'.
+
+$ inagoctl up k8s-master
+2016-04-14 16:35:23.503 | INFO     | context.Background: Succeeded to submit group 'k8s-master'.
+2016-04-14 16:36:30.600 | INFO     | context.Background: Succeeded to start group 'k8s-master'.
 ```
+
+A quick look at `kubectl version` will tell us that we're now running a 1.2.2 server.
```

Comment:
yes, I agree, actually have some in the blog post, but kinda forgot them here. will add.